packages/common/src/chains/mainnet.json

```diff
@@ -91,7 +91,7 @@
       "//_comment": "The forkHash will remain same as mergeForkIdTransition is post merge, terminal block: https://etherscan.io/block/15537393",
       "name": "merge",
       "ttd": "58750000000000000000000",
-      "block": null,
+      "block": 15537394,
```

Comment:
https://etherscan.io/block/15537394 🎉

So cool to see this getting in and have a block number and everything. 🥳🥳🥳

💯🙏

So nice to spot the various Etherscan changes between 15537393 and 15537394. 🙂